src/java.base/windows/classes/sun/nio/fs/WindowsFileStore.java

```diff
@@ -221,8 +231,7 @@ public boolean supportsFileAttributeView(String name) {
         if (name.equals("owner"))
             return supportsFileAttributeView(FileOwnerAttributeView.class);
         if (name.equals("user"))
-            return supportsFileAttributeView(UserDefinedFileAttributeView.class);
-        return false;
+            return supportsFileAttributeView(UserDefinedFileAttributeView.class); return false;
```

Comment:
Can you revert this change, it's not part of the fix and it makes this method harder to maintain.

Reply:
I did revert it before but I guess it crept back in ... sigh.